wgsl/index.bs

```diff
@@ -1003,8 +1003,23 @@ array_decoration_list
 
 array_decoration
   : STRIDE PAREN_LEFT INT_LITERAL PAREN_RIGHT
+
+ident_type_decoration_list
+  : ATTR_LEFT (ident_type_decoration COMMA)* ident_type_decoration ATTR_RIGHT
+
+ident_type_decoration
+  : ACCESS PAREN_LEFT access_type PAREN_RIGHT
+
+access_type
+  : READ
+  | READ_WRITE
 </pre>
 
+The access decoration must only appear on a type used as the store type for a
+variable in the `storage` [=storage class=]. The access decoration must not appear
+on a type of const declaration nor as the store type for variable with a
+[=storage class=] other than `storage`.
```

Comment:
Is the access qualifier  *required* on the store type for variable in `storage` storage class?  Or does it default to read_write?

Reply:
I strongly prefer it being required. Storage buffers are not child toys, they should be used with knowledge and caution.

Reply:
Added text to say they're required.